src/main/java/Command.java

```diff
@@ -0,0 +1,126 @@
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.List;
+
+public class Command {
+    protected String command;
+    protected int order;
+    protected LocalDate date;
+    protected boolean hasDate;
+    protected String time;
+    protected String type;
+    protected boolean isExit;
+    protected int priority;
+
+    public Command(String command) {
+        this.command = command;
+    }
+
+    public Command(String command, int order) {
+        this.command = command;
+        this.order = order;
+    }
+
+    public Command(String type, String command) {
+        this.type = type;
+        this.command = command;
+    }
+
```

Comment:
Since `String command` is common to all the overloaded constructors, maybe it would be good to have it first in the list of parameters?